ha/core/controllers/pcs/service_controller.py

```diff
@@ -82,3 +99,25 @@ def status(self, service: str, nodeids: list = None) -> dict:
                 status: Succeeded, Failed, InProgress
         """
         raise HAUnimplemented("This operation is not implemented.")
+
+    def clear_resources(self, node_id: str):
+        """
+        Clear resources on node.
+
+        Args:
+            nodeid (str): Private fqdn define in conf store.
+        """
+        try:
+            resources: list = []
+            output, _, _ = self._execute.run_cmd(const.LIST_PCS_RESOURCES, check_error=False)
+            if "NO resources" not in output:
+                for resource in output.split("\n"):
+                    res = resource.split(":")[0]
+                    if res != "" and res not in resources:
+                        resources.append(res)
+            for resource in resources:
+                self._execute.run_cmd(f"pcs resource clear {resource} {node_id}")
+            Log.info(f"Waiting to clear resource on node {node_id}")
+            time.sleep(20)
```

Comment:
same as comment on line 82

Reply:
Here we don't need time.sleep(). Tested on HW setup it runs fine

Reply:
In that case remove the sleep  or if sleep is needed, add the jira number (jira created for same comment above ) here as well

Reply:
sleep is removed from here, we don't need it here.